bonfire/bonfire.py

```diff
@@ -884,7 +908,7 @@ def _get_apps_config(
 
     # handle git ref/image substitutions if reference environment was provided
     if ref_env:
-        apps_config = sub_refs(apps_config, ref_env, fallback_ref_env, preferred_params=prefer)
+        apps_config = sub_refs(apps_config, ref_env, fallback_ref_env, preferred_params)
```

Comment:
Renaming 'prefer' to 'preferred_params' was supposed to go in with #333 -- not sure what happened but it was just for readability so we'll get the change in here :man_shrugging: